### PR TITLE
Russian translations improvment

### PR DESCRIPTION
**Pull request by shiche**

### CODE REVIEW

File: src/dlangui/dialogs/settingsdialog.d [hunk 1]
```diff
@@ -256,7 +256,7 @@ class ExecutableFileNameEditItem : SettingsItem {
         import dlangui.dialogs.filedlg;
         TextWidget lbl = new TextWidget(_id ~ "-label", _label);
         FileNameEditLine ed = new FileNameEditLine(_id ~ "-filename-edit");
-        ed.addFilter(FileFilterEntry(UIString.fromRaw("MESSAGE_EXECUTABLES"d), "*.exe", true));
+        ed.addFilter(FileFilterEntry(UIString.fromRaw("MESSAGE_EXECUTABLES"c), "*.exe", true));
```
Comment (and3md) [inline src/dlangui/dialogs/settingsdialog.d]: `UIString` has two functions `fromRaw()` and `fromId()`. If you want use id like `MESSAGE_EXECUTABLES` you should use `fromId()`. `UIString.fromRaw()` just shows "MESSAGE_EXECUTABLES" not "Executable files" when English is chosen.

File: src/dlangui/dialogs/filedlg.d [hunk 2]
```diff
@@ -927,7 +928,7 @@ class FileNameEditLine : HorizontalLayout {
     protected EditLine _edFileName;
     protected Button _btn;
     protected string[string] _filetypeIcons;
-    protected dstring _caption = "TITLE_OPEN_FILE"d;
+    protected dstring _caption = dtext("TITLE_OPEN_FILE"c);
```
Comment (and3md) [inline src/dlangui/dialogs/filedlg.d]: Only `UIString` can be translated. This place in code need more work.

File: src/dlangui/dialogs/filedlg.d [hunk 3]
```diff
@@ -597,9 +598,9 @@ class FileDialog : Dialog, CustomGridCellAdapter {
         _fileList.layoutWidth(FILL_PARENT).layoutHeight(FILL_PARENT);
         _fileList.resize(4, 3);
         _fileList.setColTitle(0, " "d);
-        _fileList.setColTitle(1, "COL_NAME"d);
-        _fileList.setColTitle(2, "COL_SIZE"d);
-        _fileList.setColTitle(3, "COL_MODIFIED"d);
+        _fileList.setColTitle(1, dtext("COL_NAME"c));
```
Comment (and3md) [inline src/dlangui/dialogs/filedlg.d]: I think currently this lines are unable to translate. This need a change in `StringGridWidget`. Correct me if I'm wrong. Why you use `dtext()`?

Comment (and3md) [inline src/dlangui/dialogs/filedlg.d]: OK, I found another solution. It's not ideal but for `FileDialog` should be good enough. You can just create `UIString` here from ID and get translated value: `UIString.fromId("COL_NAME"c).value`
Translation will not change when you change language when `FileDialog` is open but that will be better than no translation.

File: src/dlangui/dialogs/filedlg.d (hunk 2 above)
Comment (and3md) [inline src/dlangui/dialogs/filedlg.d]: The same like above: `UIString.fromId("TITLE_OPEN_FILE"c).value;`

**Comment by buggins** [comment]
looks good for me, too.
Merging...

**Pull request merged**